Sources/TuistGenerator/Generator/ProjectDescriptorGenerator.swift

```diff
@@ -81,6 +81,8 @@ final class ProjectDescriptorGenerator: ProjectDescriptorGenerating {
     ) throws -> ProjectDescriptor {
         logger.notice("Generating project \(project.name)")
 
+        let previousXcodeProj = try? XcodeProj(pathString: project.xcodeProjPath.pathString)
```

Comment:
I'd benchmark this one to get a sense of the performance penalty. If it's significant, I'd consider adding a generation option, something along the lines of `.preserveLaunchConfiguration`.

Reply:
I did a quick test and it takes milliseconds

Reply:
Good!